pkgs/os-specific/linux/purism/librem-ec-acpi-dkms.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchgit, kernel }:
+
+stdenv.mkDerivation rec {
+  pname = "librem-ec-acpi-dkms-${version}-${kernel.version}";
+  version = "0.9.1";
+
+  src = fetchgit {
+    url = "https://source.puri.sm/nicole.faerber/librem-ec-acpi-dkms.git";
+    rev = "9be1cc47fbe245f84adb5979c61ce74ae7f68b83";
+    sha256 = "1qnbfj60i8nn2ahgj2zp5ixd79bb0wl1ld36x3igws2f3c0f5pfi";
+  };
+
+  hardeningDisable = [ "pic" ];
+
+  KERNEL_DIR = "${kernel.dev}/lib/modules/${kernel.modDirVersion}/build";
+  INSTALL_MOD_PATH = placeholder "out";
+
+  nativeBuildInputs = kernel.moduleBuildDependencies;
+
+  buildFlags = [ "all" ];
+
+  installFlags = [ "DEPMOD=true" ];
+  enableParallelBuilding = true;
+
+  patches = [
+    ./librem-ec-acpi-dkms.patch
+  ];
+
+  meta = with lib; {
+    homepage = "https://source.puri.sm/nicole.gaerber/librem-ec-acpi-dkms";
+    maintainer = "Nicole Faerber (nicole.faerber@puri.sm)";
```

Comment:
Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add NAME`` into this PR.

Reply:
Is this field meant to be the maintainer of the nix, or the maintainer of the software package itself? Both?

Reply:
The maintainer of the nix package.

Reply:
Ok, fixed!